src/Dfe.PlanTech.Web/Views/Content/Shared/_CsHeader.cshtml

```diff
@@ -1,16 +1,19 @@
 <header role="banner">
-    <partial name="CookiesBanner/_CookieBanner" />
+    @{
+        var homeSlug = Context.User.Identity?.IsAuthenticated == true ? "/self-assessment" : "/";
```

Comment:
This is now the 5th instance of `/self-assessment` appearing in a non-test C# file. 12 if you include tests. 23rd if you include JS and TS.

Is it worth making a constant for this somewhere?